lib/answers2.rb

```diff
@@ -0,0 +1,22 @@
+### Challenge 2 - Reverse a string
+
+# Reverse a string in place. In other words, do not create a new
+# string or use other methods on the string such as `reverse`. The goal
+# of the problem is to use a loop and the string accessors to figure out which
+# values to swap for other values. Below is some sample output.
+
+# ```
+# Enter a string:
+# reverse_me
+# em_esrever
+# ```
+
+def reverse_string(string)
+  i = string.length
+  while i > 0
+    i -= 1
+    print string[i]
+  end
+end
+
+reverse_string('ivar')
```

Comment:
good but this doesn't actually create a reversed string. here is an alternative -
puts  'Enter a string:'
 reverse = gets.chomp.strip
 
 arr = reverse.chars
  for i in (0..(arr.length-1)/2)
   arr[arr.length-1-i], arr[i] = arr[i], arr[arr.length-1-i]
 end
  puts arr.join